README.md

```diff
@@ -161,10 +161,33 @@ logging config format for content details.
 
 -  Volume metadata file got deleted while removing volume from VM(placed on Esx2) which is in use by another VM(placed on Esx1) [#1191](https://github.com/vmware/docker-volume-vsphere/issues/1191). It's an ESX issue and will be available in the next vSphere release.
 -  Full volume name with format like "volume@datastore" cannot be specified in the compose file for stack deployment. [#1315](https://github.com/vmware/docker-volume-vsphere/issues/1315). It is a docker compose issue and a workaround has been provided in the issue.
--  Specifying "Datastore Cluster" name during volume creation is not supported. Datastore clusters (as a part of Storage DRS) is a VC feature and not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)
 -  Volume creation using VFAT filesystem is not working currently. [#1327](https://github.com/vmware/docker-volume-vsphere/issues/1327)
--  Volume is not shown when running ```docker volume ls``` and ```vmdkops_admin volume ls``` command after performing Storage vMotion of VM when volume is attached to VM. It is an expected behavior since Storage vMotion changes the location and name of attached vmdk files.
-We only see this when trying to do Storage vMotion with volume attached. So avoid Storage vMotion when volume is attached. [#1618](https://github.com/vmware/docker-volume-vsphere/issues/1618)
+-  Volume of filesystem type ext4dev cannot be attached to a container. ext4dev needs to be mounted as ext4. [#1617](https://github.com/vmware/docker-volume-vsphere/issues/1617)
+
+# Interop with vSphere features
+
+**SvMotion, XvMotion**
+
+- Volume is not shown when running ```docker volume ls``` and ```vmdkops_admin volume ls``` command after performing Storage vMotion or XvMotion of a VM when volume is attached to the VM. It is an expected behavior since Storage vMotion or XvMotion changes the location and name of attached vmdk files.
+We only see this when trying to do Storage vMotion or XvMotion with volume attached. So please avoid Storage vMotion or XvMotion when volume is attached. [#1618](https://github.com/vmware/docker-volume-vsphere/issues/1618)
+
```

Comment:
We can clarify the issue but after mentioning that s/xvmotion isn't supported for the below reasons (that the migration causes vDVS volumes to also be migrated out of the datastore location and hence causing the plugin to loose access to the volume .....